packages/ui/src/i18n/dictionaries/authenticator/de.ts

```diff
@@ -99,7 +99,7 @@ export const deDict: AuthenticatorDictionary = {
   'Sign Up with Google': 'Mit Google registrieren',
   'Forgot Password': 'Passwort vergessen',
   'Have an account? ': 'Schon registriert? ',
-  'Incorrect username or password':
+  'Incorrect username or password.':
     'Falscher Benutzername oder falsches Passwort',
```

Comment:
Changing this key will break consumers that may be using it, even if it is erroneous. To avoid breaking potential consumers would provide the correct key and leave the incorrect key:
```suggestion
  // erroneous key left in place to avoid breaking consumers
  'Incorrect username or password':
    'Falscher Benutzername oder falsches Passwort',
  'Incorrect username or password.':
    'Falscher Benutzername oder falsches Passwort',
```

The same approach should be supplied to the other files updated in this PR.

Reply:
Thank you! I modified the translations.